ANNOUNCE.rst

```diff
@@ -5,7 +5,34 @@ Announcing bcolz 0.9.0
 What's new
 ==========
 
-#XXX version-specific blurb XXX#
+This is mostly a smallish feature and bugfix release. One large topic
+was implementing 'addcol' and 'delcol' to properly handle on-disk
+tables.  When adding a disk-backed carray to a ctable, you can now use
+the boolean keyword argument 'move' to either move the data (True) or
+copy it (False) into the directory of the original ctable. To avoid
+unexpected behaviour, copying also copies the Python object, whereas a
+moving the data doesn't. When removing columns with 'delcol' you can
+specify the keyword argument 'keep' if you wish to keep the data in case
+of disk-backed carrays. Additionally, ctable now supports an 'auto_flush'
+keyword that makes it flush to disk automatically after any methods that
+may write data.
+
```

Comment:
The sentence above is good but this is an announcement, and as such, I tend to make a shorter summary and then redirect to the release notes for people interested in more details.